test/tools/provisioner-cli/main.go

```diff
@@ -97,6 +99,7 @@ func main() {
 
 	if *action == "uploadimage" {
 		log.Info("Uploading PodVM Image...")
+		os.Setenv("TEST_PROVISION", "yes")
```

Comment:
You'll want to set this, if the action is the `uploadimage` action, before getting the cloud provisioner as the get function initializes the cloud provisioner.